docs/whole-tutorial.md

```diff
@@ -400,16 +400,77 @@ $ ick run --patch
 As written, our rule would run for any Python project, but it will run when
 *any* file in the project changes.  We can be smarter than this since there are
 just two files we care about.  We might read both, and might write one and
-delete the other, so we specify them as both input and output:
+delete the other, so we specify them as inputs:
 
 ```toml
 inputs = ["pyproject.toml", "isort.cfg"]
-outputs = ["pyproject.toml", "isort.cfg"]
 ```
 
-It's safe to omit `inputs` and `outputs`, but the rule will run more often than
+On `project` and `repo` scoped rules, it's safe to omit `inputs`, since ick will pull in every file by default. However, the rule will run more often than
 it needs to.
 
+## Best Practices on How to Access Files in Rules
+
+Under the hood, `ick` takes all the files in your `inputs`, puts them into a temporary directory, and passes their names to the command line. This means they are all accessible via `sys.argv[1:]` and direct system paths like `Path("filename")`. Thanks to this functionality, the best practices on which method to use are very flexible. They largely revolve around what your rule would like to do. 
```

Comment:
`sys.argv[1:]` is Python.  You could be more expansive: "This means they are accessible as command-line arguments, `sys.argv[1:]` in Python."